.github/workflows/release.yaml

```diff
@@ -459,8 +484,12 @@ jobs:
       - name: Download WASM artifacts
         uses: actions/download-artifact@3be87be14a055c47b01d3bd88f8fe02320a9bb60 # v2.0.10
         with:
-          name: ${{ needs.build-release-wasm-libs.outputs.upload-name }}
-          path: ${{ needs.build-release-wasm-libs.outputs.upload-path }}
+          path: ${{ runner.temp }}/wasm_assets
```

Comment:
Does this rely on `runner.temp` to not change?

Reply:
the [github actions docs](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#runner-context) state that this temp dir is emptied at the beginning and end of each job, so i would assume that it stays populated between the steps within one job.